docs/rules/lines-around-comment.md

```diff
@@ -172,6 +172,55 @@ function foo(){
 }
 ```
 
+### allowClassStart
+
+Examples of **correct** code for this rule with the `{ "beforeLineComment": true, "allowClassStart": true }` option:
```

Comment:
Added incorrect examples with `allowClassStart: false`. It's not as relevant for incorrect examples with `allowClassStart: true` as it's making the check less strict.